tests/unit/remote/test_s3.py

```diff
@@ -0,0 +1,40 @@
+from unittest import TestCase
+
+from dvc.remote.s3 import RemoteS3
+
+
+class TestRemoteS3(TestCase):
```

Comment:
Looks great! I'm really sorry for not mentioning this before, but we use pytest-tests now, could you please convert this one?